modules/bigquery/README.md

```diff
@@ -57,3 +58,21 @@ so that all dependencies are met.
 | self\_link | The self\_link URI for the destination bigquery dataset |
 
 <!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+
+## Customer Managed Encryption Key permission
+
+Your project's BigQuery service account `bq-{{PROJECT_NUMBER}}@bigquery-encryption.iam.gserviceaccount.com` must have `roles/cloudkms.cryptoKeyEncrypterDecrypter` to use this feature.
+
+```hcl
+
+data "google_bigquery_default_service_account" "bq_sa" {
+  project = "gcp_dataset_project_id"
+}
+
+resource "google_kms_crypto_key_iam_member" "bq_key_iam" {
+  crypto_key_id = "kms_key_id"
+  role          = "roles/cloudkms.cryptoKeyEncrypterDecrypter"
+  member        = "serviceAccount:${data.google_bigquery_default_service_account.bq_sa.email_address}"
+}
```

Comment:
Thanks for adding this info. Wondering if it makes sense to embed this in the module? We already have `kms_key_id` as the optional input?

Reply:
I was thinking about it but trying to avoid additional permission needed to use this module as keys are typically created and managed by security team and the team deploying log sink may not have `cloudkms.cryptoKey.setIamPolicy` permission on project or key.
Although the service account email address for BQ & storage service agent follows a well-known format, the service account is created on-demand by calling data source `google_bigquery_default_service_account` for bq and `google_storage_project_service_account`. This is the reason I added example code in readme.

Reply:
Yeah makes sense. I also had some reservations adding the need for`cloudkms.cryptoKey.setIamPolicy` for this feature but also thought log-exports are most likely to to be setup by security teams. We can add this later if there is demand.